package.json

```diff
@@ -40,9 +42,13 @@
     "sanitize-html": "^2.10.0",
     "tailwind-merge": "^1.12.0",
     "tailwindcss": "^3.3.1",
-    "wagmi": "^0.12.7"
+    "usehooks-ts": "^2.9.1",
+    "viem": "^1.10.12",
+    "wagmi": "^1.4.1"
```

Comment:
wagmi v1 introduced quite a lot of changes, are you sure to bump version? In case of, they still maintain 0.12.x 😓

Reply:
It wasn't used in many places and I made the changes to accommodate the new `bigint` types returned.